.travis.yml

```diff
@@ -15,20 +15,6 @@ install:
 
 jobs:
   include:
-    - stage: test
```

Comment:
Given that Node 12 was release on 2019-04-23 and is now current - this should be updated to test against Node 12 rather than deleted

Reply:
I'd suggest to add a a block for node 12 just after the one for node 10 and see what happens.

Btw @jason-fox do you know which node 12 hasn't set yet as current in https://github.com/nodejs/Release table? (I'm looking rigth now and I see "Pending").

Reply:
Fixed by https://github.com/telefonicaid/iotagent-json/pull/400/commits/1f31e8c7d6c6939a3ec1d042a34752711de46599

Reply:
@fgalan - according to https://travis-ci.org/telefonicaid/iotagent-node-lib/jobs/526754611 Travis is using `12.1.0`

```console
nvm install 12
```
```text
Downloading and installing node v12.1.0...
Downloading https://nodejs.org/dist/v12.1.0/node-v12.1.0-linux-x64.tar.xz...
Computing checksum with sha256sum
Checksums matched!
Now using node v12.1.0 (npm v6.9.0)
```

Reply:
> Btw @jason-fox do you know which node 12 hasn't set yet as current in https://github.com/nodejs/Release table? (I'm looking rigth now and I see "Pending").

Now it reads "Current release". It seems they changed 4 days ago (https://github.com/nodejs/Release/pull/440/files).